deploy/podman/flightctl-ui/flightctl-ui-config/init.sh

```
@@ -0,0 +1,70 @@
+    sed -n -E "s/^[[:space:]]*${key}:[[:space:]]*[\"']?([^\"'#]+)[\"']?.*$/\1/p" "$VALUES_FILE"
+}
+
+# Check if values file exists
+if [ ! -f "$VALUES_FILE" ]; then
+  echo "Error: Values file not found at $VALUES_FILE"
+  exit 1
+fi
+
+# Extract base values from values.yaml
+BASE_DOMAIN=$(extract_value "baseDomain")
+SRV_CERT_FILE=$(extract_value "srvCertFile")
```

Shellcheck (0.10.0)

[warning] SRV_CERT_FILE appears unused. Verify use (or export if used externally).

(SC2034)